src/zzz_od/application/hollow_zero/lost_void/operation/lost_void_move_by_det.py

```
@@ -424,27 +461,87 @@ def get_entry_target(self, frame_result: DetectFrameResult) -> MoveTargetWrapper
         判断是否被困
         @return:
         """
+        visible_target_list = self.get_visible_targets(frame_result)
         if self.last_target_result is None or new_target is None:
-            self.same_target_times = 0
+            self._reset_stuck_status()
+            self.last_visible_target_list = visible_target_list
             return None
 
-        if self.last_target_result.leftest_target_name != new_target.leftest_target_name:
+        all_visible_targets_static, static_reason = self.is_same_visible_target_list(
```

Ruff (0.15.18)

[warning] Unpacked variable `static_reason` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)